desktop/panel/database.test.js

```diff
@@ -162,13 +181,14 @@ for (const subprocess of RUNNERS) {
                 expect(new Date(v[0].date)).toStrictEqual(
                   new Date('2021-01-01')
                 );
-              } else {
+              } else if (t.query.startsWith('SELECT')) {
                 expect(v).toStrictEqual([
                   { name: 'Kate', age: 9, city: 'San Juan' },
                   { name: 'Bake', age: 10, city: 'Toronto' },
                 ]);
+              } else {
+                expect(v).toBe('');
```

Comment:
This case just shouldn't be possible. So I'd `throw` here so we make sure it never happens

Reply:
Hmm, like I want to test the insert to make sure the doesn't error with ODBC. I guess we could ignore that case then.

Reply:
Oh I'm sorry I misunderstood. What you had was ok then

Reply:
Could you just add a comment saying why this happens

Reply:
Yeah, it's due to the error generated here when it's binding columns https://github.com/alexbrainman/odbc/blob/9c9a2e61c5e2c1a257a51ea49169fc9008c51f0e/odbcstmt.go#L134.